processes/fulfillURLManifest.py

```diff
@@ -80,34 +79,35 @@ def update_manifest(self, metadataObject, bucketName, currKey):
         if metadataJSON != metadataJSONCopy:
             try:
                 fulfillManifest = json.dumps(metadataJSON, ensure_ascii = False)
-                return self.putObjectInBucket(
-                    fulfillManifest, currKey, bucketName
+                return self.s3Client.put_object(Bucket=bucketName, Key=currKey, \
+                                Body=fulfillManifest, ACL= 'public-read', \
+                                ContentType = 'application/json'
                 )
             except ClientError as e:
                 logging.error(e)
 
-    def linkFulfill(self, metadataJSON):
+    def linkFulfill(self, metadataJSON, counter):
```

Comment:
what's the counter param in these functions?

Reply:
The counter parameter is to keep track of the number of links that are updated in the manifest file. The counter should be at least 4 before hitting the fulfillFlagUpdate method since all manifests we generate have a similar format of the types of links provided.